### DISCUSSION TITLE
Making things private!

### DISCUSSION THREAD

**Discussion by marijo-blazevic** [AlgebraStudentCollab/START-HERE#13] [Announcements]
I talked with @JosipKustan about making repos private till we are ready, safe to make them public. For now only members we trust/know should have access. 

Opinions? - @tsucevic  @lcabraja  @JosipKustan

**Comment by lcabraja** [comment]
fine with that for the most part just two things:
 - The notes repo can stay public
 - and I have no idea how forking works with private repos, we might need a pro for the organization or for ourselves which we have

**Reply by marijo-blazevic** [reply]
- Agree!
- Forking private [repos](https://stackoverflow.com/a/68246412) is still possible for non paid [accounts](https://stackoverflow.com/a/9629558)

**Reply by marijo-blazevic** [reply]
START-HERE can also stay public

**Comment by tsucevic** [comment]
I don't see a reason as to why we would do that, but I am fine with it happening

**Reply by marijo-blazevic** [reply]
My reasoning is to better be safe than sorry

**Reply by lcabraja** [reply]
we could ask Bele for input, off the record

or we could spin this
anyone who has the exact same source code well you know who and where they got it from
which is might only be a problem when we use code from stack overflow

**Comment by marijo-blazevic** [comment]
And if we plan to upload any materials (presentations, etc.) we should ask for permission..  The rules are not clear.

Who will ask Bele? - when asking, say we plan to do it, don't say we did it ;) 😅

**Reply by JosipKustan** [reply]
I can be the spokesman. Ill be discreet :)
